app/assets/stylesheets/application.scss

```diff
@@ -46,15 +48,15 @@ a {
   padding: 1rem 1rem;
 }
 
-.container-albums-show {
```

Comment:
Why remove this? It's needed for the album show page...

Reply:
It's just removed for now, still WIP. Was messing up our carousel somehow.